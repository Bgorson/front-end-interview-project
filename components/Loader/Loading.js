import ClipLoader from 'react-spinners/ClipLoader';
import React from 'react';

class Loading extends React.Component {

    render() {
        return (
            <div className="spinner">
                <ClipLoader
                    size={350}
                    color={'#123abc'}
                />
            </div>
        );
    }
}
export default Loading;
