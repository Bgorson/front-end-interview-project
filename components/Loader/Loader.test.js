import renderer from 'react-test-renderer';
import Loading from './Loading';

it('should render correctly', () => {
    const component = renderer.create(<Loading/>).toJSON();

    expect(component).toMatchSnapshot();
});