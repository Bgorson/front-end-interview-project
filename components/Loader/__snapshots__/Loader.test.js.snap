// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render correctly 1`] = `
<div
  className="spinner"
>
  <div
    className="css-1xmnk2x"
  />
</div>
`;
